data/classes/fx.json

```diff
@@ -1,17 +1,26 @@
 {
   "name": "fx",
+  "javadoc": [
+    "A position of a block in 3 dimensions."
+  ],
   "fields": [
     {
       "name": "a",
-      "descriptor": "Lcom/mojang/serialization/Codec;"
+      "descriptor": "Lcom/mojang/serialization/Codec;",
+      "javadoc": [
+        "The codec for a {@code BlockPos}."
+      ]
     },
     {
       "name": "b",
       "descriptor": "Lfx;"
     },
     {
       "name": "e",
-      "descriptor": "Lorg/apache/logging/log4j/Logger;"
+      "descriptor": "Lorg/apache/logging/log4j/Logger;",
+      "javadoc": [
+        "The logger for the class."
+      ]
```

Comment:
Comment is not useful. It's expected knowledge to know what a logger for a class is which violates the standards.

Reply:
Removed javadoc to comply with standards.